frontend/src/scenes/billing/Billing.tsx

```diff
@@ -138,16 +138,21 @@ export function Billing(): JSX.Element {
             )}
 
             {billing?.trial ? (
-                <LemonBanner type="info" className="mb-2">
+                <LemonBanner type={billing.trial.type === 'autosubscribe' ? 'info' : 'warning'} className="mb-2">
```

Comment:
Might be cool to make this more fun with a hedgehog or something, make it more of a celebration

Reply:
Added hedgehogs, star if on autosubscribe, builder if not. Updated PR description with new screenshots.